internal/worker/uniter/relation/statemanager.go

```diff
@@ -127,7 +127,7 @@ func (m *stateManager) initialize() error {
 		return errors.Trace(err)
 	}
 	m.relationState = make(map[int]State, len(unitState.RelationState))
-	if m.logger.IsTraceEnabled() {
+	if m.logger.IsLevelEnabled(logger.TRACE) {
```

Comment:
And also one day change all x.logger fields to x.log fields to be in line with the log variable/arg naming.